locales/en.yml

```diff
@@ -1,6 +1,6 @@
 en:
   theme_metadata:
-    description: "A simple, beautiful theme that improves the out of the box experience for Discourse sites."
+    description: "A modern theme designed for communities that want a beautiful community out of the box."
```

Comment:
Personal opinion:

> A modern theme designed for communities that want a beautiful community out of the box.

Community is repeated twice. I think 'look' might be nice

> A modern theme designed for communities that want a beautiful look out of the box.